### DISCUSSION TITLE
Is it possible to combine an ObservableList and an Observable?

### DISCUSSION THREAD

**Discussion by Guidemarcus** [reactivemarbles/DynamicData#701] [Q&A]
Hi,

I am new to this library and I am trying to figure out how it works. On my project, I have on ObservableList and an observable. The list represent a set of items and the observable represents a selected filter tag. 

I would like to combine the two to have only one ObservableList that would update if either my list or my filter is updated.

Is this possible?

Visual representation:
ObservableList<ItemType> + Observable<Filter> = ObservableList<ItemType>
On this, I would use the Filter function that DynamicData offers

**Comment by RolandPheasant** [comment]
In this case, you should just use the filter functionality.

There are several examples here: https://github.com/RolandPheasant/DynamicData.Snippets/tree/master/DynamicData.Snippets/Filter
